index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Link to font family -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css"/>
+    <title>Document</title>
+</head>
+<body>
+    <header class="site-header">
+        
+        <a href="#" > Laxmi Basnet</a>
+        <nav class="nav-menu">
+            <img src="./images/Vector.png" alt="Vector image ">
+        </nav>
+
+    </header>
+    <section class="container">
+        
+            <div class="text">
+                <h1> Hey There. I'm Laxmi</h1>
+        
+        
+                <h3>I'm a Software Developer.</h3>
+            
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+             
+            </div>
+            <div class="icon">
+            <ul >
+                <li><a href="#"><img src="./images/icon1.png" alt=""></a></li>
+                <li><a href="#"><img src="./images/icon2.png" alt=""></a></li>
+                <li><a href="#"><img src="./images/icon4.png" alt=""></a></li>
+                <li><a href="#"><img src="./images/icon3.png" alt=""></a></li>
+                <li><a href="#"><img src="./images/icon5.png" alt=""></a></li>
+            </ul>
+        </div>
+        
+            
+        
```

Comment:
- [ ] It would be better if you wrap these lines with a `main` tag in order to make your code Semantic like you used the `header` tag in your header section. To see more about Semantic HTML, click this [link](https://www.freecodecamp.org/news/semantic-html5-elements/) 😊.